packages/react-composites/src/composites/CallComposite/components/LocalDeviceSettings.tsx

```diff
@@ -250,8 +250,9 @@ export const LocalDeviceSettings = (props: LocalDeviceSettingsType): JSX.Element
             {onResolveVideoEffectDependency && (
               <DefaultButton
                 iconProps={{ iconName: 'ConfigurationScreenVideoEffectsButton' }}
-                styles={effectsButtonStyles(theme)}
+                styles={effectsButtonStyles(theme, props.cameras.length === 0)}
```

Comment:
we use two different variables to control `disabled` here
`props.cameras.length === 0` and `!cameraPermissionGranted`

These should presumably always be the same

Reply:
I will check for both since one I think is looking at the permissions API since its only used in beta atm